logistic_requisition/model/stock.py

```diff
@@ -46,3 +47,15 @@ def name_get(self, cr, user, ids, context=None):
             name = "%s - (%s)" % (rs.code, rs.name)
             res += [(rs.id, name)]
         return res
+
+    @api.model
+    def name_search(self, name, args=None, operator='ilike', limit=100):
+        """
+        Allows to search incoterms by code
+        """
+        if name:
+            recs = self.search([('code', operator, name)], limit=limit)
+            if recs:
+                return recs.name_get()
```

Comment:
Don't return this, but aggregate to other possible values from the name search, respecting limit variable. The best option for this is to combine all criteria in the search.